doc/RecordingControl.xml

```diff
@@ -2119,13 +2124,151 @@ secfrac = "." 1*6DIGIT</programlisting>
+aligned(8) class AsymmetricKeySystemHeaderBox extends FullBox('pssh', version=1, flags=0)
+{
+  unsigned int(8)[16] SystemID;
+  unsigned int(32) KID_count = 1;
+  unsigned int(8)[16] KID;
+  unsigned int(32) EncryptedKeyEntryCount;
+  for (i=1; i <= EncryptedKeyEntryCount; i++) {
+    unsigned int(16) CertificateThumbprintAlgorithm;
+    unsigned int(32) CertificateThumbprintSize;
+    unsigned int(8)[CertificateThumbprintSize] CertificateThumbprint;
+    unsigned int(16) EncryptionVersion;
+    unsigned int(32) EncryptionDataSize;
+    if (EncryptionVersion == 1) {
+      unsigned int(8)[EncryptionDataSize] EncryptedKey;
+    } else if (EncryptionVersion == 2) {
+      unsigned int(16) HpkeKem;
+      unsigned int(16) HpkeKdf;
+      unsigned int(16) HpkeAead;
+      unsigned int(8)[EncapsulatedSharedSecretSize] EncapsulatedSharedSecret;
+      unsigned int(16) EncryptedKeySize
+      unsigned int(8)[EncryptedKeySize] EncryptedKey;
+    }
+
+  }
+}
+            ]]></programlisting>
+          </section>
+          <section>
+            <title>Semantics</title>
+            <para><literal>Version</literal> shall be '1'.</para>
+            <para><literal>Flags</literal> shall be '0'.</para>
+            <para><literal>SystemID</literal> shall be <literal>'ca774354-6f98-41b6-b17a-b15f7bbf678a'</literal>.</para>
+            <para><literal>KID_count</literal> shall be '1'.</para>
+            <para><literal>KID</literal> is the generated UUID by the device. This <literal>KID</literal> should be the same in all segments until the symmetric key changes.</para>
+            <para><literal>DataSize</literal> is the size in bytes of all the other fields present in this box following this field.</para>
+            <para><literal>EncryptedKeyEntryCount</literal> Number of entries containing encryption information required to decrypt the symmetric key. Shall be 1 or more.</para>
+            <para><literal>CertificateThumbprintAlgorithm</literal> defines the algorithm used to compute the thumbprint of the certificates. Those values are defined in the Security Baseline specification</para>
+            <para><literal>CertificateThumbprintSize</literal> Size of the <literal>CertificateThumbprint</literal> field.</para>
+            <para><literal>CertificateThumbprint</literal> Thumbprint of the certificate used to encrypted the symmetric key.</para>
+            <para><literal>EncryptionVersion</literal> defines the encryption strategy used to encrypt the symmetric key for this certificate.</para>
+            <para><literal>EncryptionDataSize</literal> defines the number of bytes used by the following bytes for this certificate. The next certificates, if any, will start after this number of bytes.</para>
+            <para><literal>EncryptedKey</literal> The symmetric key (identified by <literal>KID</literal>) used for frame encryption, encrypted using the public key of the certificate according to the encryption version.</para>
```

Comment:
RSA encryption needs a paragraph on padding. Suggest to require state of the art OAEP.